src/cmd-run

```diff
@@ -161,12 +164,19 @@ EOF
     }
 }
 EOF
+    if [ "${ignition_version}" = "2.2.0" ]; then
+        ign_validate="true"
```

Comment:
Could still validate the config before downgrading it to v2 though, no?

Reply:
Oh yeah, but we'd have to put a build up of `ignition` spec 2 somewhere.  And in this case we know it's valid; what we really want to do is validate in the converter I think.